tools/checkBehaviourTree.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
```

Comment:
Not sure if this should be committed since it only checks for that one port: `error_code_id`, I think the name of the file is too general and doesn't describe what this does

Reply:
Its a useful debugging tool to check, but not sure this needs to be in Nav2 now that you've shown them all complete